src/hardware/mouse/mouse_config.cpp

```diff
@@ -263,26 +263,26 @@ static void config_init(Section_prop &secprop)
 	prop_str->Set_values(list_capture_types);
 	prop_str->Set_help(
 	        "Choose a mouse control method:\n"
-	        "  onclick:   Capture the mouse when clicking any button in the window.\n"
+	        "  onclick:   Capture the mouse when clicking any button in the window (default).\n"
 	        "  onstart:   Capture the mouse immediately on start. Might not work correctly\n"
 	        "             on some host operating systems.\n"
 	        "  seamless:  Let the mouse move seamlessly; captures only with middle-click or\n"
 	        "             hotkey. Seamless mouse does not work correctly with all the games,\n"
 	        "             Windows 3.1x can be made compatible with a custom mouse driver.\n"
 	        "  nomouse:   Hide the mouse and don't send input to the game.\n"
-	        "For touchscreen control use the seamless method.\n");
+	        "For touchscreen control, use 'seamless'.");
 
 	prop_bool = secprop.Add_bool("mouse_middle_release", always, true);
-	prop_bool->Set_help("If true, middle-click will release the captured mouse, and also\n"
-	                    "capture when seamless.");
+	prop_bool->Set_help("If enabled, middle-click will release the captured mouse, and also\n"
+	                    "capture when seamless (enabled by default).");
 
 	prop_multi = secprop.AddMultiVal("mouse_sensitivity", only_at_start, ",");
 	prop_multi->Set_help(
-	        "Default mouse sensitivity. 100 is a base value, 150 is 150% sensitivity, etc.\n"
+	        "Default mouse sensitivity (100,100 by default). 100 is the base value, 150 is 150% sensitivity, etc.\n"
 	        "Negative values reverse mouse direction, 0 disables the movement completely.\n"
 	        "The optional second parameter specifies vertical sensitivity (e.g. 100,200).\n"
```

Comment:
Maybe this portion can be reworded a bit more:

```
"Mouse sensitivity for the horizontal and vertical axes,
as a percentage (100,100 by default). Negative values invert
the axis, zero disables it. Providing only one value sets
sensitivity for both axes."
```

Reply:
That's a lot better, changing it.